crates/ruff_cli/Cargo.toml

```diff
@@ -59,6 +60,7 @@ walkdir = { version = "2.3.2" }
 
 [dev-dependencies]
 assert_cmd = { version = "2.0.8" }
+indoc = "2.0.1"
```

Comment:
Nit: we tend to use the expanded version everywhere (I don't know why, but it is the convention).

Reply:
tbh i'm a big fan of indoc everywhere, but yes consistency